architecture/additional_concepts/service_catalog.adoc

```diff
@@ -0,0 +1,321 @@
++
+.Example `Broker` Specification
+----
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Broker
+metadata:
+  name: BestDataBase
+spec:
+  url: http://bestdatabase.example.com
+----
+
+Service Class::
+Also synonymous with "service" in the context of the service catalog, a _service
+class_ is a type of managed service offered by a particular broker. Each time a
+new broker resource is added to the cluster, the service catalog controller
+connects to the corresponding broker server to obtain a list of service
+offerings. A new `ServiceClass` resource is automatically created for each.
++
+.Example `ServiceClass` Specification
+----
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: ServiceClass
+metadata:
+  name: smallDB
+  brokerName: BestDataBase
+  plans: [...]
+----
+
+Service Plan::
+A _service plan_ is one type of variant of a service class. For example, a
+service class may expose a set of plans that offer varying degrees of
+quality-of-services (QoS), each with a different cost associated with it.
+
+Service Instance::
+A _service instance_ is a provisioned instance of a service class. These are
+created by cluster users who wish to make a new, distinct instance of some type
+of managed service to make that available for use by one or more of their
+applications. When a new `Instance` resource is created, the service catalog
+controller connects to the appropriate broker server and instructs it to
+provision the service instance.
++
+.Example `Instance` Specification
+----
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Instance
+metadata:
+  name: my_db
+spec:
+  serviceClassName: smallDB
+----
+
+Credentials::
+_Credentials_ are information needed by an application to communicate with a
+service instance.
+
+Service Binding::
+A _service binding_ is a link between a service instance and an application.
+These are created by cluster users who wish for their applications to reference
+and use a service instance. Upon creation, the service catalog controller
+creates a Kubernetes secret containing connection details and credentials for
+the service instance. Such secrets can be mounted into pods as usual.
++
+.Example `Binding` Specification
+----
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Binding
+metadata:
+  name: myBinding
+spec:
+  secretName: mySecret
+  <pod_selector_labels>
+----
+
+[[service-catalog-provided-brokers]]
+== Provided Service Brokers
+
+{product-title} provides the following service brokers for use with the service
+catalog.
+
+[NOTE]
+====
+Because the service catalog is currently Technology Preview, the following
+service brokers are also currently Technology Preview.
+====
+
+[[service-catalog-template-service-broker]]
+=== Template Service Broker
+
+The _template service broker_ gives the service catalog visibility into the
+xref:../../dev_guide/templates.adoc#using-the-instantapp-templates[default Instant App and Quickstart templates] that have shipped with {product-title}
```

Comment:
Not sure on the precise text but I'd specifically mention the concept that the TSB can make available as a service anything for which a template has been written - whether by us, by the administrator and/or consumer of OpenShift, or a 3rd party